doc/source/conf.py

```diff
@@ -32,12 +33,25 @@ def get_wheelhouse_assets_dictionary():
     assets_context_runners = ["ubuntu-latest", "windows-latest", "macos-latest"]
     assets_context_python_versions = ["3.9", "3.10", "3.11", "3.12"]
     if get_version_match(__version__) == "dev":
-        # Just point to the latest version
-        assets_context_version = json.loads(
-            requests.get(
+
+        # Try to retrieve the content three times before failing
+        content = None
+        for _ in range(3):
+            response = requests.get(
                 "https://api.github.com/repos/ansys/pyansys-geometry/releases/latest"
-            ).content
-        )["name"]
+            )
+            if response.status_code == 200:
+                content = response.content
+                break
+            else:
+                print(f"Failed to retrieve the latest release. Retrying...")
+                time.sleep(2)
+
+        if content is None:
+            raise requests.exceptions.RequestException("Failed to retrieve the latest release")
```

Comment:
```suggestion
            raise requests.exceptions.RequestException("Failed to retrieve the latest release.")
```
Message strings should conclude with punctuation.